dxr/plugins/js/filters.py

```diff
@@ -0,0 +1,31 @@
+from jinja2 import Markup
+from dxr.filters import QualifiedNameFilterBase
+from dxr.plugins.js.refs import PLUGIN_NAME
+
+
+class _QualifiedNameFilter(QualifiedNameFilterBase):
+    lang = PLUGIN_NAME
+
+
+class PropFilter(_QualifiedNameFilter):
+    name = 'prop'
+    is_identifier = True
+    description = Markup('Property definition filter: <code>prop:foo</code>')
```

Comment:
Let's stick "JavaScript" in the description somewhere for now. Eventually, I want to badge the filters in the menu according to which languages they support and then also badge the trees in the Switch Tree menu according to which language plugins are enabled.